### PR TITLE
Feature/er 760 pl headers

### PR DESCRIPTION
**Pull request by dgcollard**

### CODE REVIEW

File: assets/scss/base/_type.scss [hunk 1]
```diff
@@ -1,8 +1,16 @@
-
 html {
   font-size: $font-size-base;
 }
 
 body {
   font-size: 1rem;
 }
+
```
Comment (heylookalive) [inline assets/scss/base/_type.scss]: Can't we set this via $headings-font-weight?

Comment (dgcollard) [inline assets/scss/base/_type.scss]: Good idea

**Comment by openbook** [comment]
code reviewed, marking as ready for test

**Comment by heylookalive** [comment]
Ergh we need to sort out the font sizing and baseline stuff - our heading sizes are wack.

**Pull request merged**